xdsl/mlir_converter.py

```diff
@@ -2,9 +2,9 @@
 
 ensure_mlir_module_loaded()
 
-from xdsl import _mlir_module as mlir
+from xdsl import _mlir_module as mlir  # type: ignore
```

Comment:
What type issues are you ignoring here? I wouldn't have expected a type ignore here :sweat_smile: